openshift/start-minishift.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 
 startMinishift() {
-  minishift start
+  minishift start --ocp-tag=latest
```

Comment:
Hmmmm, why did you add this? This could make the whole experience a moving target. If I install a minishift version, say for openshift 3.10, when 3.11 comes out it'll try that and keep moving forward. Things might break... I'd leave up to the user to make a conscious decision on which Minishift version to use.

Reply:
hey, @galderz I knew adding this option would be debatable and can revert it. the reason I added it though is that - in docs - we advise users to go with the latest version of OpenShift. also, according to PM, 3.11 is the minimum version for 7.3. if you think it's better, we can keep it as part of the requirements in the readme and let users make the conscious decision as you said.

Reply:
also, I found that there is a config option for this: `openshift-version` so it would be better to include in the setup script as `minishift config set openshift-version`, not that it matters. I've removed that option anyway. cheers.

Reply:
Ok, let's leave it out for now.